lib/core/include/irods/client_connection.hpp

```diff
@@ -174,6 +355,18 @@ namespace irods::experimental
                                const std::string_view _username,
                                const std::string_view _zone) -> void;
 
+        auto only_connect(const std::string_view _host,
+                          const int _port,
+                          const std::string_view _username,
+                          const std::string_view _zone) -> void;
+
+        auto only_connect(const std::string_view _host,
+                          const int _port,
+                          const std::string_view _proxy_user,
+                          const std::string_view _proxy_zone,
+                          const std::string_view _username,
+                          const std::string_view _zone) -> void;
```

Comment:
All `std::string_view`'s should probably be changed to `const std::string&` for safety purposes.

Reply:
Updated.